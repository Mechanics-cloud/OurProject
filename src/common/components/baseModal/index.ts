export * from './BaseModal'
export * from './SimpleModal'
