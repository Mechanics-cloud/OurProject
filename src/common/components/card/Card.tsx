import { ComponentProps } from 'react'

type Props = ComponentProps<'div'>

export const Card = ({className, children, ...props}: Props) => {
  return (
    <div className={`bg-dark-500 rounded-sm border border-dark-300 px-6 py-4 text-light-100 ${className}`}
         {...props}>
      {children}</div>
  )
}