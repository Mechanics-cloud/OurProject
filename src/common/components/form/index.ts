export * from './FormCheckbox'
export * from './FormTextField'
