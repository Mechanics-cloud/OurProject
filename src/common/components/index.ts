export * from './asyncComponent'
export * from './avatar'
export * from './baseModal'
export * from './button'
export * from './calendar'
export * from './card'
export * from './checkbox'
export * from './datePicker'
export * from './fullscreenLoader'
export * from './goTopButton'
export * from './header'
export * from './header/langSelect'
export * from './like'
export * from './linkWithIcon'
export * from './loader'
export * from './menu'
export * from './overlay'
export * from './pagination'
export * from './popover'
export * from './radioGroup'
export * from './reCaptcha'
export * from './registrationResult'
export * from './scrollbar'
export * from './select'
export * from './sideBar'
export * from './skeleton'
export * from './tabs'
export * from './textUnfolding'
export * from './textarea'
export * from './textfield'
export * from './toast'
export * from './tooltip'
export * from './typography'
export * from './userMiniLink'
