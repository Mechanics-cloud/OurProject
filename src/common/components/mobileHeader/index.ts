export * from './MobileHeader'
//comment for run lint