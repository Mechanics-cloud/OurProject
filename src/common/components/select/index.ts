export { CustomSelect } from './CustomSelect'
export * from './Select'
