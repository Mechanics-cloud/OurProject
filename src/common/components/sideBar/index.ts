export * from './LoadingSidebar'
export * from './SideBar'
