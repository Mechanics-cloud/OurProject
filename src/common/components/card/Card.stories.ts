import type { Meta, StoryObj } from '@storybook/react';
import { Card } from '@/common/components/card/Card'

const meta = {
  title: 'Component/Card',
  component: Card,
  parameters: {
    layout: 'centered',
  },
  tags: ['autodocs'],

} satisfies Meta<typeof Card>;

export default meta;
type Story = StoryObj<typeof meta>;

export const BasicCard: Story = {
  args: {
    children: 'Basic Card'
  },
};
