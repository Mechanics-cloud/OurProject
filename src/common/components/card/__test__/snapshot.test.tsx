import { Card } from '@/common/components/card'
import { render } from '@testing-library/react'

it('renders card unchanged', () => {
  const { container } = render(<Card />)

  expect(container).toMatchSnapshot()
})
