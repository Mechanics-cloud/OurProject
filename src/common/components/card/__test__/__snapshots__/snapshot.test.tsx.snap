// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders card unchanged 1`] = `
<div>
  <div
    class="bg-dark-500 rounded-sm border border-dark-300 px-6 py-4 text-light-100 undefined"
    data-testid="card-element"
  />
</div>
`;
